specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -14692,6 +14757,39 @@
         }
       }
     },
+    "RecoveryPlanProviderSpecificDetails": {
+      "description": "Recovery plan provider specific details.",
+      "type": "object",
+      "properties": {
+        "instanceType": {
+          "description": "Gets the Instance type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "discriminator": "instanceType"
+    },
+    "RecoveryPlanProviderSpecificInput": {
+      "description": "Recovery plan provider specific input base class.",
+      "type": "object",
+      "properties": {
+        "instanceType": {
+          "description": "Gets the Instance type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "canWrite": {
+          "description": "Gets a value indicating whether this Newtonsoft.Json.JsonConverter can write JSON.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "canRead": {
+          "type": "boolean",
+          "readOnly": true
+        }
+      },
+      "discriminator": "instanceType"
```

Comment:
If you use `instanceType` as the discriminator, it must be a required field.

Reply:
Just like the CI errors said.

Reply:
I have merged with latest code and sematic error s are not there, please take a look thanks